### PR TITLE
Fix missing attribute error

### PR DESCRIPTION
**Pull request by MykhailoLiutov**
Add the missing attribute for `boxBackgroundMode `

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -11,7 +11,8 @@ buildscript {
         classpath 'com.android.tools.build:gradle:3.6.0'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
```
Comment (mottljan) [inline build.gradle]: I think this should be removed

File: lib.properties [hunk 2]
```diff
@@ -1,12 +1,15 @@
-VERSION_NAME=1.1.1
-VERSION_CODE=3
-GROUP=cz.ackee.ui
-BINTRAY_REPO=ui
-BINTRAY_ORGANISATION=ackeecz
+VERSION_NAME=1.1.2
+VERSION_CODE=4
+GROUP=io.github.ackeecz.ui
```
Comment (mottljan) [inline lib.properties]: group does not contain the `ui` suffix

File: lib.properties [hunk 3]
```diff
@@ -1,12 +1,15 @@
-VERSION_NAME=1.1.1
-VERSION_CODE=3
-GROUP=cz.ackee.ui
-BINTRAY_REPO=ui
-BINTRAY_ORGANISATION=ackeecz
+VERSION_NAME=1.1.2
```
Comment (mottljan) [inline lib.properties]: 2.0.0. This release contains breaking changes

File: lib.properties [hunk 4]
```diff
@@ -1,12 +1,15 @@
...
+VERSION_CODE=4
```
Comment (mottljan) [inline lib.properties]: just for your info - this probably does nothing :D usually we keep 1 there, not sure why it is even there

File: ui/build.gradle [hunk 5]
```diff
@@ -32,12 +32,8 @@ dependencies {
 }
 
 ext {
-    bintrayName = 'TextInputLayout'
-    publishedArtifactId = 'TextInputLayout'
-
-    libraryName = 'TextInputLayout'
-    libraryDescription = 'Google Material Component without the hint cut out.'
+    PUBLISH_ARTIFACT_ID = 'TextInputLayout'
+    POM_DESCRIPTION = 'TextInputLayout'
```
Comment (mottljan) [inline ui/build.gradle]: Please provide the previous proper description here

File: build.gradle (hunk 1 above)
Comment (MykhailoLiutov) [reply build.gradle]: Removed https://github.com/AckeeCZ/TextInputLayout/pull/12/commits#:~:text=%E2%99%BB%EF%B8%8F,and%20fix%20version

File: lib.properties (hunk 2 above)
Comment (MykhailoLiutov) [reply lib.properties]: Removed https://github.com/AckeeCZ/TextInputLayout/pull/12/commits#:~:text=%E2%99%BB%EF%B8%8F,and%20fix%20version

File: lib.properties (hunk 3 above)
Comment (MykhailoLiutov) [reply lib.properties]: Updated https://github.com/AckeeCZ/TextInputLayout/pull/12/commits#:~:text=%E2%99%BB%EF%B8%8F,and%20fix%20version

File: lib.properties (hunk 4 above)
Comment (MykhailoLiutov) [reply lib.properties]: Changed to 1
https://github.com/AckeeCZ/TextInputLayout/pull/12/commits#:~:text=%E2%99%BB%EF%B8%8F,and%20fix%20version

File: ui/build.gradle (hunk 5 above)
Comment (MykhailoLiutov) [reply ui/build.gradle]: Sorry, it got lost somehow, updated
https://github.com/AckeeCZ/TextInputLayout/pull/12/commits#:~:text=%E2%99%BB%EF%B8%8F,and%20fix%20version

**Pull request merged**